esrally/chart_generator.py

```diff
@@ -1587,7 +1587,7 @@ def bulk_tasks(self):
         task_names = []
         for task in self.track.find_challenge_or_default(self.challenge).schedule:
             for sub_task in task:
-                if sub_task.operation.type == track.OperationType.Bulk.to_hyphenated_string():
+                if track.OperationType.Bulk.to_hyphenated_string() in sub_task.operation.type:
```

Comment:
I think this warrants at least a comment and maybe even an info message to the terminal on lenient matches.